diff --git a/node_modules/primereact/components/calendar/Calendar.js b/node_modules/primereact/components/calendar/Calendar.js
index acf0bec..432cd2a 100644
--- a/node_modules/primereact/components/calendar/Calendar.js
+++ b/node_modules/primereact/components/calendar/Calendar.js
@@ -1753,9 +1753,9 @@ function (_Component) {
   }, {
     key: "renderDayNames",
     value: function renderDayNames(weekDays) {
-      var dayNames = weekDays.map(function (weekDay) {
+      var dayNames = weekDays.map(function (weekDay, index) {
         return _react.default.createElement("th", {
-          key: weekDay,
+          key: index,
           scope: "col"
         }, _react.default.createElement("span", null, weekDay));
       });
